android/src/main/java/com/joinflux/flux/segment/SegmentPlugin.java

```diff
@@ -26,24 +26,27 @@ public void initialize(PluginCall call) {
             return;
         }
         String key = call.getString("key");
+        Boolean trackLifecycle = call.getBoolean("trackLifecycle", false);
         if (key == null) {
             call.reject("Write key is required to initialize plugin");
             return;
         }
 
         Context context = this.getContext();
-        Builder builder = new Analytics.Builder(context, key);
```

Comment:
Why was this code moved back into the UI thread? (i.e. this [commit](https://github.com/joinflux/capacitor-segment/commit/62555b325a770bc77d70e3c4ec7f98797b8434b0) seems to have been reverted)?

Reply:
Thanks for picking this up, this was not meant to be re-added, will fix this